CHANGELOG.asciidoc

```diff
@@ -142,6 +159,8 @@ types, because it is only expected to be useful for coming OTel Bridge work.
 [float]
 ===== Features
 
+[float]
+===== Bug fixes
```

Comment:
This looks like accidental CHANGELOG-merge-cruft.

Reply:
had to move the CHANGELOG up to the new unreleased section but I'm _pretty sure_ all the cruft's been removed
<img width="1292" alt="Screen Shot 2022-06-08 at 2 38 19 PM" src="https://user-images.githubusercontent.com/184372/172722595-ad6a7140-a108-4960-9161-15e2e7b68246.png">
.